util/fibers/fiber2.h

```diff
@@ -109,6 +109,12 @@ uint64_t FiberLongRunSumUsec() noexcept;
 // It is advised to call this function when a program starts.
 void SetDefaultStackResource(PMR_NS::memory_resource* mr, size_t default_size = 64 * 1024);
 
+// Returns the total size for worker fibers for the current thread.
+size_t WorkerStackSize();
```

Comment:
```suggestion
// Returns the total stack size for worker fibers for the current thread.
size_t WorkerFibersStackSize();
```

Reply:
Maybe it's also worth clarifying that maybe only part of this size is actually allocated and present in rss